app/manifest/v2/firefox.json

```diff
@@ -4,20 +4,5 @@
       "id": "webextension@metamask.io",
       "strict_min_version": "78.0"
     }
-  },
-  "permissions": [
-    "storage",
-    "unlimitedStorage",
-    "clipboardWrite",
-    "http://localhost:8545/",
-    "https://*.infura.io/",
-    "https://*.codefi.network/",
-    "https://chainid.network/chains.json",
-    "https://lattice.gridplus.io/*",
-    "activeTab",
-    "tabs",
```

Comment:
I think we still need to keep this one for firefox old browser support bug ~~

Reply:
I suggest we solve that seperately. If we can find a way to solve that bug without adding the permission, lets do that. We could wait until we have agreed on the final approach before merging this PR.

I can rebase my other PR on top of this PR's branch and we can discuss that alternative solution there

Reply:
That's perfect 🙏